### PR TITLE
Set russian language

### PR DESCRIPTION
**Pull request by akprof2000**
Write translate file for Russian language

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/command-line-api?pullRequest=1381) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/command-line-api?pullRequest=1381) before we can accept your contribution.<br/><br/>:x: akprof2000 [sign now](https://cla.dotnetfoundation.org/dotnet/command-line-api?pullRequest=1381)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/command-line-api?pullRequest=1381) it.</sub>

File: src/System.CommandLine/Properties/xlf/Resources.ru.xlf [hunk 1]
```diff
@@ -1,216 +1,216 @@
...
       <trans-unit id="CommandExpectsOneArgument">
         <source>Command '{0}' expects a single argument but {1} were provided.</source>
-        <target state="new">Command '{0}' expects a single argument but {1} were provided.</target>
+        <target state="new">Команда '{0}' ожидает единственный аргумент но {1} предоставлено.</target>
         <note />
       </trans-unit>
       <trans-unit id="CommandNoArgumentProvided">
         <source>No argument was provided for Command '{0}'.</source>
-        <target state="new">No argument was provided for Command '{0}'.</target>
+        <target state="new">Нет аргументов для команды '{0}'.</target>
```
Comment (adamsitnik) [inline src/System.CommandLine/Properties/xlf/Resources.ru.xlf:32]: nit: according to google translate it means "No argument for command". We are missing the "was provided"

File: src/System.CommandLine/Properties/xlf/Resources.ru.xlf [hunk 2]
```diff
@@ -1,216 +1,216 @@
...
       <trans-unit id="ErrorReadingResponseFile">
         <source>Error reading response file '{0}': {1}</source>
-        <target state="new">Error reading response file '{0}': {1}</target>
+        <target state="new">Ошибка при чтение файла ответа '{0}': {1}</target>
         <note />
       </trans-unit>
       <trans-unit id="ExceptionHandlerHeader">
         <source>Unhandled exception: </source>
-        <target state="new">Unhandled exception: </target>
+        <target state="new">Необработанная ошибка: </target>
```
Comment (adamsitnik) [inline src/System.CommandLine/Properties/xlf/Resources.ru.xlf:71]: nit: According to google translate `ошибка` means "error". Would "исключение" be a better name for exception?

File: src/System.CommandLine/Properties/xlf/Resources.ru.xlf [hunk 3]
```diff
@@ -1,216 +1,216 @@
...
       <trans-unit id="HelpUsageCommandTitle">
         <source>[command]</source>
-        <target state="new">[command]</target>
+        <target state="new">[команда]</target>
         <note />
       </trans-unit>
       <trans-unit id="HelpUsageOptionsTitle">
         <source>[options]</source>
-        <target state="new">[options]</target>
+        <target state="new">[опция]</target>
```
Comment (adamsitnik) [inline src/System.CommandLine/Properties/xlf/Resources.ru.xlf:146]: We need plural (option**s**)

```suggestion
        <target state="new">[Опции]</target>
```

**Review by adamsitnik — APPROVED** [review]
The PR LGTM, thank you for your contribution @akprof2000

There is a merge conflict and the source repo has been deleted. I'll try to find a workaround to it to get it merged.

File: src/System.CommandLine/Properties/xlf/Resources.ru.xlf (hunk 3 above)
Comment (adamsitnik) [reply src/System.CommandLine/Properties/xlf/Resources.ru.xlf:146]: @KalleOlaviNiemitalo great catch!

**Comment by adamsitnik** [comment]
I am sorry, I've tried solving the merge conflicts but it seems impossible because of the fact the the repo was removed.

**Pull request closed without merging**